tests/integration/test_helpers.go

```diff
@@ -49,7 +49,8 @@ const (
 	manifestsDirectory        = "manifests/"
 	testingAppFile            = "testing-app.yaml"
 	globalCommonResourcesFile = "global-commons.yaml"
-	istioJwtApiruleFile       = "istio-jwt-strategy.yaml"
+	istioJwtApiRuleFile       = "istio-jwt-strategy.yaml"
+	istioJwtApiRuleScopesFile = "istio-jwt-scopes.yaml"
```

Comment:
This file is only used in `istio_jwt_scopes_test.go`so it might be better to have it inside of the helper as there is no benefit in doing this.